### DISCUSSION TITLE
HowTo: move statusbar below header

### DISCUSSION THREAD

**Discussion by fedoranvar** [sxyazi/yazi#1486] [1. Q&A]
good day !

want to achieve this: 
![image](https://github.com/user-attachments/assets/a71a4455-453e-4e76-82ee-394d7f708bbf)

tried like in `no-statusbar` plugin :
```lua
local function setup()
	local tab_lt = Tab.layout
	local status_lt = Status.layout
	local header_lt = Header.layout



	Tab.layout = function(self, ...)
		self._area = ui.Rect { x = self._area.x, y = self._area.y + 1, w = self._area.w, h = self._area.h }
		return tab_lt(self, ...)
	end
	Status.layout = function(self, ...)
		self._area = ui.Rect { x = tab_lt._area.x, y = tab_lt._area.y - 1, w = self._area.w-3, h = self._area.h }
		return status_lt(self, ...)
	end
end

return { setup = setup }

```

but no luck

**Comment by AnirudhG07** [comment]
You just swap Status and Header. 
In your main "init.lua", what you do is take the https://github.com/sxyazi/yazi/blob/shipped/yazi-plugin/preset/components/header.lua and https://github.com/sxyazi/yazi/blob/shipped/yazi-plugin/preset/components/status.lua, put them in your main init.lua, and swap the names Header and Status.
[init.zip](https://github.com/user-attachments/files/16625888/init.zip)

An example of swapping them.

**Reply by AnirudhG07** [reply]
You can try experimenting with it. If not, you can confirm what you want exactly, if i didnt get it right. 

If you also mean, no header and put status in header, then maybe you can try using `no-statusbar` plugin combined with above tip to make something work

**Comment by fedoranvar** [comment]
managed to resolve this

![image](https://github.com/user-attachments/assets/984afa90-b5c0-4f23-b7fc-1a98b13eeb75)


by changing places in `Root:build` and `Root:Layout` functions of `yatline`

![image](https://github.com/user-attachments/assets/9a0ff291-0a73-4e08-93a1-b71780cb0c5b)

**Reply by AnirudhG07** [reply]
Oh right, thats also a nice way to do it. Thanks for sharing it here!
